zeitschrift-fur-digitale-geisteswissenschaften.csl

```diff
@@ -0,0 +1,389 @@
+        <if type="book">
+          <group>
+            <choose>
+              <if variable="volume">
+                <text prefix=". " term="volume" form="short" suffix=".&#160;"/>
+                <text variable="volume"/>
+              </if>
+            </choose>
+          </group>
+        </if>
+      </choose>
+      <text prefix=" / " macro="secondary-contributors"/>
+    </group>
+  </macro>
+  <macro name="title-short">
+    <group delimiter=". ">
+      <choose>
+        <if type="book">
+          <text variable="title" form="short"/>
+        </if>
+        <else>
+          <text variable="title" form="short"/>
+        </else>
+      </choose>
+    </group>
+  </macro>
+  <macro name="container">
+    <choose>
+      <if type="post-weblog">
+        <text prefix=" In: " variable="container-title"/>
+      </if>
+    </choose>
+    <choose>
+      <if type="chapter">
+        <choose>
+          <if variable="container-author editor" match="any">
+            <text variable="container-title" prefix="In: " suffix="."/>
+            <text macro="container-contributors"/>
+          </if>
+        </choose>
+        <choose>
+          <if variable="number-of-volumes">
+            <text prefix=". " variable="number-of-volumes" suffix=" "/>
+            <text term="volume" form="long"/>
+          </if>
+        </choose>
+        <text prefix=" / " macro="secondary-container-contributors"/>
+      </if>
+      <else-if type="article-journal article-magazine article-newspaper" match="any">
+        <text prefix="In: " variable="container-title"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="collection">
+    <text variable="collection-title"/>
+    <choose>
+      <if is-numeric="collection-number">
+        <group delimiter=" ">
+          <text prefix=" " variable="collection-number"/>
+        </group>
+      </if>
+      <else>
+        <text prefix=", " variable="collection-number"/>
+      </else>
+    </choose>
+    <choose>
+      <if variable="collection-editor">
+        <text prefix=" / " term="editor" form="verb-short" text-case="lowercase" suffix=". "/>
+        <text macro="secondary-collection-contributors"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <number variable="edition" form="ordinal"/>
+        <text prefix="&#160;" term="edition" form="short" suffix="."/>
+      </if>
+      <else-if variable="edition">
+        <text variable="edition"/>
+        <text prefix=" " term="edition" form="short" suffix="."/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="issued">
+    <choose>
+      <if type="book">
+        <text prefix=". " suffix=" Bde" variable="number-of-volumes"/>
+        <text prefix=". " macro="edition"/>
+        <text prefix=" (" suffix=")" variable="collection-title"/>
+        <text prefix=". " variable="publisher-place"/>
+        <text prefix=". " suffix=". " variable="note"/>
```

Comment:
Wofuer ist das? Wir sehen die Nutzung von `note` in Standad Stilen nur sehr ungern

Reply:
Ups, das war mal für die interne Ausgabe einer internen ID zum Zweck der leichteren Überprüfung aus welchem Artikel die Angaben stammen. Habe ich entfernt, danke!